examples/wave-eager.py

```diff
@@ -54,7 +55,8 @@ def bump(actx, discr, t=0):
 def main():
     cl_ctx = cl.create_some_context()
     queue = cl.CommandQueue(cl_ctx)
-    actx = PyOpenCLArrayContext(queue)
+    actx = PyOpenCLArrayContext(queue,
+        allocator=cl_tools.MemoryPool(cl_tools.ImmediateAllocator(queue)))
```

Comment:
It'll be pretty easy to forget that, and as a result get rotten performance. Could you submit a patch to meshmode that produces a warning if a `PyOpenCLArrayContext` is created without an allocator? (where the warning can be silenced by passing `allocator=False`)

Reply:
Done here: https://github.com/inducer/meshmode/pull/34